src/DynamoCoreWpf/UI/Themes/Modern/OutPorts.xaml

```diff
@@ -93,6 +96,15 @@
                 </Border.Style>
             </Border>
 
+            <!--  A small marker on the left of the port, indicating whether it's been honored  -->
+            <Rectangle x:Name="PortValueMarker"
+                       Grid.Column="1"
+                       Height="27px"
+                       VerticalAlignment="Center"
+                       Fill="{Binding PortValueMarkerColor, UpdateSourceTrigger=PropertyChanged}"
+                       IsHitTestVisible="False"
+                       SnapsToDevicePixels="True" />
```

Comment:
What are these for?

Reply:
This represents the output bar
![State5_OutPutBar](https://user-images.githubusercontent.com/89042471/161116448-3efe5eae-c647-4129-a4a5-500d6fb3aa46.jpg)

Reply:
Once the comment updated here, LGTM

Reply:
Comment updated. Thanks!